string robot_id
float64 target_height
float64 x
float64 y
float64 theta
float64 sigma_x
float64 sigma_y
float64 sigma_theta